config/configmap/inferenceservice.yaml

```diff
@@ -185,3 +185,14 @@ data:
         "cpuRequest": "100m",
         "cpuLimit": "1"
     }
+  defaults: |-
+    {
+        "request": {
+            "cpu": "1",
+            "memory": "2Gi"
+        },
+        "limit": {
+            "cpu": "1",
+            "memory": "2Gi"
+        },
```

Comment:
No trailing comma here

Reply:
Hi @markwinter , thanks for the review!  I removed the trailing comma, but the test still fails. Please, can you point me where to look for the test logs ? I look at the  `https://argo.kubeflow-testing.com/workflows/kubeflow-test-infra/kubeflow-kfserving-presubmit ...` link, from the test `Details` -> `more info `link. There I can see the pods, the `run-e2e-test` fails, but in its logs the only error is `Waiting for KFServing started ... error: no matching resources found`